dirs/dirs.go

```diff
@@ -552,27 +552,41 @@ func SetRootDir(rootdir string) {
 	LocaleDir = filepath.Join(rootdir, "/usr/share/locale")
 	ClassicDir = filepath.Join(rootdir, "/writable/classic")
 
-	opensuseTWWithLibexec := func() bool {
+	opensuseFlavorWithLibexec := func() bool {
 		// XXX: this is pretty naive if openSUSE ever starts going back
 		// and forth about the change
+
+		if release.DistroLike("opensuse-slowroll") {
+			// Slowroll does not need further checks, it has used /usr/libexec
+			// since the start
+			return true
+		}
+
 		if !release.DistroLike("opensuse-tumbleweed") {
+			// Leap and the like, are there other
```

Comment:
Looks like truncated?
```suggestion
			// Leap and the like, are there others?
```

Reply:
There are more derivatives, like microos